tests/test_config.py

```
@@ -0,0 +1,345 @@
+        """Test Crawl4AI advanced features configuration."""
+        config = CrawlerrSettings(
+            crawl_adaptive_mode=False,
+            crawl_confidence_threshold=0.9,
+            crawl_top_k_links=10,
+            crawl_url_seeding=False,
+            crawl_score_threshold=0.5,
+            crawl_virtual_scroll=False,
+            crawl_scroll_count=30,
+        )
+
+        assert config.crawl_adaptive_mode == False
```

Ruff (0.12.2)

Avoid equality comparisons to `False`; use `not config.crawl_adaptive_mode:` for false checks

Replace with `not config.crawl_adaptive_mode`

(E712)